app/src/containers/Event/EventDetails.js

```diff
@@ -84,9 +85,14 @@ class EventDetails extends React.Component {
               </ListGroup>
             </Tab>
             <Tab eventKey={3} title='Gifts'>
-              <Link className='list-group-item' to={'/events/' + id + '/gifts'}>
-                  Gifts
+              <div>
+                <PageHeader className='text-center'> Gifts </PageHeader>
+              </div>
+              <Link className='list-group-item' to={`/events/${id}/gifts`}>
+                  Sample Gifts
               </Link>
+              { this.props.children }
+              <Comments eventId={id} giftId={1} author={this.props.owner} />
```

Comment:
Temporary Comments are attached to the test component so far which will be replaced by adding a working one